native/src/components/TtsPlayer.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components/native'
+
+import { CloseIcon, PauseIcon, PlaybackIcon, PlayIcon } from '../assets'
+import Icon from './base/Icon'
+import IconButton from './base/IconButton'
+import Pressable from './base/Pressable'
+import Text from './base/Text'
+
+const StyledTtsPlayer = styled.View<{ $isPlaying: boolean }>`
+  background-color: ${props => props.theme.colors.grayBackgroundColor};
+  border-radius: 28px;
+  width: ${props => (props.$isPlaying ? '90%' : '80%')};
+  display: flex;
+  flex-direction: ${props => (props.$isPlaying ? 'column' : 'row')};
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  position: absolute;
+  bottom: 5px;
+  min-height: 93px;
+  gap: ${props => (props.$isPlaying ? '0px;' : '10px')};
```

Comment:
```suggestion
  gap: ${props => (props.$isPlaying ? '0px;' : '20px')};
```

To increase the gap between play and close button